adapters/adtrgtme/adtrgtme.go

```diff
@@ -18,7 +18,7 @@ type adapter struct {
 }
 
 // Builder builds a new instance of the Adtrgtme adapter for the given bidder with the given config.
-func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
+func Builder(_ openrtb_ext.BidderName, config config.Adapter, _ config.Server) (adapters.Bidder, error) {
```

Comment:
I guess it is ok to pass name instead of `_`. You should not have to change this and let's be consistent.